SU-HSDO/suhumsci
# HSD8-411 custom francestanford stylesheet

# READY FOR REVIEW

# Summary
- Add configuration for custom France-Stanford stylesheet and theme option

# Needed By (Date)
- End if sprint

# Urgency
- N/A (though it is a prereq for other tickets)

# Steps to Test

1. Pull this branch
2. Verify configuration files look good
3. Verify that there is now a France Stanford theme setting at `admin/appearance/settings/su_humsci_theme`

# Affected Projects or Products
- suhumsci (theme)
- France-Stanford

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/HSD8-411

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
